clusterloader2/testing/network/1_1_ratio_override.yaml

```diff
@@ -0,0 +1,2 @@
+NUMBER_OF_SERVERS: 1
```

Comment:
If you look into our tests, we're moving towards prefixing those vars with CL2_ (then you basically can parametrize tests without even having overrides).

So please prefix those variables with CL2_
(here and in other override files).

Reply:
yes modified.